app/assets/javascripts/darkswarm/services/authentication_service.js.coffee

```diff
@@ -17,6 +15,19 @@ Darkswarm.factory "AuthenticationService", (Navigation, $modal, $location, Redir
       @selectedPath = path || @selectedPath
       Navigation.navigate @selectedPath
 
+    initialTab: ->
```

Comment:
Some documentation would help understanding the code, thanks!

Reply:
I refactored this a bit because the logic was getting complex. I've added some comments in the code now to clarify. These methods just choose which template will be used for the modal, and which tab will be selected when it opens (the layout and tab selection is different in /registration pages).